theme/templates/resource-landing-page/subject.html

```diff
@@ -51,5 +59,6 @@ <h3>Subject Keywords</h3>
         </div>
     </div>
 </div>
-
+<link href="{{ STATIC_URL }}css/vue-bootstrap-typeahead.0.2.6.css" rel="stylesheet">
+<script type="text/javascript" src="{{ STATIC_URL }}js/vue-bootstrap-typeahead.0.2.6.min.js"></script>
```

Comment:
Could you indicate where this script is being used?

Reply:
It's used by the mover script. My undestanding is that we don't notate things like this, where My Resources area has a bunch of implied dependencies and developers are required to manually trace through and understand what is available in the final JS context, after Django templates and includes are all processed...

Reply:
In that case we might need a comment next to this import since it is not immediately obvious that it serves as a dependency for Mover.

Reply:
I can do this. In general, stuff really needs to be cleaned up in this regard in pre-existing code. I have a few ideas, though not sure how much you already have plans to refactor, anyway.

Reply:
Fixed in 81118c7c7